modules/arm_plugin/README.md

```diff
@@ -0,0 +1,212 @@
+
+# [OpenVINO™ Toolkit](https://01.org/openvinotoolkit) - ARM CPU plugin
+
+OpenVINO™ ARM CPU plugin is developed in order to enable deep neural networks inference on ARM CPUs, using OpenVINO™ API. The plugin uses [ARM Compute Library\*] as a backend.
+
+## Supported Platforms
+OpenVINO™ ARM CPU plugin is supported and validated on the following platforms: 
+
+Host  | OS
+------------- | -------------
+Raspberry Pi 4 Model B   | Debian 10.3
+
+## Distribution
+OpenVINO™ ARM CPU plugin is not included into Intel® Distribution of OpenVIVO™. To use the plugin, it should be built from source code.
+
+## How to build
+### Approach #1: build OpenCV, OpenVINO and the plugin using pre-configured Dockerfile (cross-compiling)
+OpenVINO™ and ARM CPU plugin could be built in Docker container for [32-bit](Dockerfile.RPi32) and [64-bit](Dockerfile.RPi64) Debian:
+
+1. Clone `openvino_contrib` repository:
+```
+git clone --recurse-submodules --single-branch --branch=master https://github.com/openvinotoolkit/openvino_contrib.git 
+```
+2.  Go to plugin directory:
+ ```
+cd openvino_contrib/modules/arm_plugin
+```
+3. Build the plugin in Docker container:
+```
+docker image build -t arm-plugin -f Dockerfile.RPi32 .
+```
+4. Export the archive with artifacts to the current directory:
+```
+docker run -ti -v $PWD:/remote arm-plugin cp ./OV_ARM_package.tar.gz /remote
```

Comment:
- `docker run --rm`
- do you need "root"-owned files? what is about files permissions?

Reply:
Added `--rm` option to `docker run` command.
Owner of copied `OV_ARM_package.tar.gz` archive is root but file permissions are 644 which allows to read the archive by non-root accounts.